source/fundamentals/collations.txt

```diff
@@ -188,22 +208,27 @@ which **does not** specify a collation:
    :end-before: end findOneAndUpdate without collation
    :dedent: 4
 
-Since "Gunter" is the first result in the sorted results when using a binary
-collation, the operation does not update any documents. Consider the same
-operation with a collation specified with the locale set to
+Since "Gunter" is the first sorted result when using a binary collation, none
+of the documents come lexically before and match the ``$lt`` comparison
+operator in the query document. As a result, the operation does not update any
+documents.
```

Comment:
yes, this is exactly what I was hoping to see. Definitely makes the "why" much clearer than the previous version, fantastic job!